app/api/routers/ask_gpt.py

```diff
@@ -5,18 +5,27 @@
 from app.utils.anonymize import anonymize_text
 
 load_dotenv()
-ask_router=APIRouter()
-openai.api_key=os.getenv("OPENAI_API_KEY")
+ask_router = APIRouter()
+openai.api_key = os.getenv("OPENAI_API_KEY")
+
 
 @ask_router.get('/ask')
-async def ask(question:str,text:str|None):
-    anonymized_text= anonymize_text(text)
-    resp=openai.ChatCompletion.create(
+async def ask(question: str, text: str | None):
```

Comment:
I can see that you are using a linter to vscode or pycharm :)

Reply:
always very sharp remarks! xd